crates/core/src/client/helpers.rs

```diff
@@ -329,10 +314,10 @@ pub fn create_default_transaction_receipt() -> TransactionReceipt {
 /// # Errors
 ///
 /// TODO: add error case message
```

Comment:
do we need to update this?

Reply:
Made [an issue](https://github.com/kkrt-labs/kakarot-rpc/issues/197) for this (Not so much the TODO but think we need to implement `TryFrom`).